docs/parent-child-model-for-multiple-layers-services.md

```diff
@@ -0,0 +1,17 @@
+# Parent-Child Model for Multiple Layers Services
+
+Services like MySQL and MS SQL are not single-layer services -- they have database management system (DBMS) and database. To satisfy the user requirement to create databases on a shared DBMS, OSBA brings in the _Parent-Child Model_. Typically, the DBMS services are in the parent service category, and the database services are in the child service category.
+
+## Parent Service
+
+For a parent service, `alias` is always one of the required provisioning parameters. It is the correlative key for a parent service instance to engage with child service instances. For easy memo, it is recommended to give the same value to the service instance name.
+
+## Child Service
+
+For a child service, `parentAlias` is always one of the required provisioning parameters. It should be specified an existing `alias` of a parent service instance. A child service instance can only be created successfully in this way.
+
+## Notes
```

Comment:
One thing may be confusing to users, and it can be added to somewhere in the doc:
- On instance level, parent-child is a "one-to-many" relationship. That is, for a parent instance, it can have several child instance, but for a child instance, it has and only has one parent instance.
- On service level, parent-child is a "many-to-many" relationship. That is, a parent service has several valid child service, and a child service has several valid parent service.
For example, service A can be parent of service C,D; service B can be parent of service D,E. Note here both service A and B can be parent service of service D. However, for an instance of service D, it can only have one parent instance, either an instance of service A or an instance of service B is OK, but it can't have two parent instance.

Reply:
If it is abstract to users, we may add this statement to MSSQL module and use specific service name. After all, only MSSQL module has "many-to-many" parent-child relationship on service level currently.